docs/src/configuration.md

```diff
@@ -81,35 +102,59 @@ The following implementations should work:
 - [Microsoft MPI](https://docs.microsoft.com/en-us/message-passing-interface/microsoft-mpi)
 - [IBM Spectrum MPI](https://www.ibm.com/us-en/marketplace/spectrum-mpi)
 
-If the implementation is changed, you will need to re-run `Pkg.build("MPI")`.
+#### Advanced options
 
-### [Environment variables](@id environment_variables)
+```@doc
+MPI.use_system_binary
+```
+You can use the argument `mpiexec` to provide the the name (or full path) of the MPI launcher executable. The default is
+`mpiexec`, but some clusters require using the scheduler launcher interface (e.g. `srun`
+on Slurm, `aprun` on PBS). If the MPI library has an uncommon name you can provide it in `library_names`.
+In case ABI detection fails you can provide a manual choice,
+but also open an issue so that the auto-detection can be improved. `export_prefs=true` can be used to copy the preferences into the `Project.toml` instead of creating a
```

Comment:
```suggestion
In case ABI detection fails you can provide a manual choice (either `MPICH` or `OpenMPI` or `MicrosoftMPI`),
but also open an issue such that the automatic detection can be improved.
`export_prefs=true` can be used to copy the preferences into the `Project.toml` instead of creating a
```
I think these three are the only ones supported, right? Further, why `export_prefs` and not `export_preferences`? Ok, that's probably a personal style question, I try to avoid abbreviations where not necessary. Finally, I think we should state a reason why anyone would want to use `export_prefs`, i.e., what are the possible advantages.

Reply:
You are missing MPI trampoline.

`export_prefs` is inherited from Preferences.jl we just pass through the keyword argument.

Export preferences puts it into the Project.toml so it is one less file to copy around. Other than that there shouldn't be a difference

Reply:
I have updated the suggestion